client/controllers/events/eventIncidentReports.coffee

```diff
@@ -139,6 +139,10 @@ Template.eventIncidentReports.onRendered ->
       @updatePlot(groups)
       return
 
+  @autorun =>
+    EventIncidents.find(Template.instance().data.filterQuery.get())
+    $('tr.details').remove()
```

Comment:
Could we move this up to the `autorun` on line 122 or should we keep it separate?